service/src/main/kotlin/fi/espoo/evaka/occupancy/Occupancy.kt

```diff
@@ -750,10 +752,10 @@ WHERE sn.placement_id = ANY(:placementIds)
 
     val absences =
         if (type == OccupancyType.REALIZED)
-            this.createQuery("SELECT child_id, date FROM absence WHERE child_id = ANY(:childIds) AND :period @> date AND NOT absence_type = ANY(:nonAbsences)")
+            this.createQuery("SELECT child_id, date, care_type FROM absence WHERE child_id = ANY(:childIds) AND :period @> date AND NOT absence_type = :presence")
                 .bind("childIds", childIds)
                 .bind("period", period)
-                .bind("nonAbsences", AbsenceType.nonAbsences.toTypedArray())
```

Comment:
don't know if some renaming would help here, `financeNonAbsences`? 🤷‍♂️

Reply:
Maybe, but since it's not currently used anyway, I just wanted to delete it. I'll let the absences rewrite solve it.